khmer/_khmermodule.cc

```diff
@@ -1628,8 +1628,13 @@ static PyObject* new_hashtable(PyObject * self, PyObject * args)
     if (kcounting_obj == NULL) {
         return NULL;
     }
+    try {
+        kcounting_obj->counting = new CountingHash(k, size);
+    } catch (const std::bad_alloc&) {
+        PyErr_SetString(PyExc_MemoryError,"Out of Memory");
```

Comment:
Ah, I was thinking of `PyErr_SetNone(PyExc_MemoryError);` Turns out this was because I had already done much of this already; see: https://github.com/ged-lab/khmer/issues/510#issuecomment-75154752